install.sh

```
@@ -7,4 +7,4 @@ git submodule sync
 git submodule update
 
 # for react-native
-npm install react-native@0.27.2
+npm install react-native@0.27.2 react@15.1.0 --silent
```

It doesn't silence errors, just reduces the ouput.